Software-Engineering.md

```diff
@@ -142,13 +142,13 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I have a deep understanding of Octopus products and our strategy
 - I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
 - I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
-- My contributions leave our Engineering team, and Octopus as a whole, in a stronger strategic position
+- My contributions are strategic and take our Engineering team, and Octopus as a whole, to the next level
```

Comment:
Can we clarify what "next level" means?

Reply:
I agree this is vague. 
That being said, this seems to be one of the terms that people don't have a problem to understand. I've used it multiple times and it didn't cause any confusion. In other words, I think people should not have a problem to assess whether an idea is taking Octopus to the next level or not.

I would like to make it more concrete but I'm not sure how.  
Do you have any suggestions?

Reply:
> Do you have any suggestions?

Yes. When I think "take to the next level", I interpret that as meaning "leave in a stronger strategic position" 😄

Reply:
This is a PR for a branch so we can evolve it in the PSE branch.

Reply:
> Yes. When I think "take to the next level", I interpret that as meaning "leave in a stronger strategic position

The same level of vagueness :)